api/v1alpha1/accesslogging_types.go

```diff
@@ -19,12 +19,39 @@ type ProxyAccessLog struct {
 type ProxyAccessLogSetting struct {
 	// Format defines the format of accesslog.
 	Format ProxyAccessLogFormat `json:"format"`
+	// Filters defines the extensions filters of accesslog.
+	// +notImplementedHide
+	Filters []ProxyAccessLogFilter `json:"filters,omitempty"`
```

Comment:
thoughts on ?
```
matches:
- "response.code >= 400"
- "(connection.mtls && request.headers['x-log-mtls'] == 'true') || request.url_path.contains('v1beta3')"
```

Reply:
if we plan to support CEL only, it's fine.

Reply:
looks like I was able to convince @guydc https://github.com/envoyproxy/gateway/issues/3667#issuecomment-2187363606 :) 

also should the noun be `filters` or `matches`